components/codeChecker/massPreview/CodeCheckerMassPreview.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div class="w-full max-w-[490px] flex flex-col gap-4">
+    <CodeCheckerMassPreviewCanvas
+      :assets="assets"
+      :previews="PREVIEWS_AMOUNT" />
+
+    <CodeCheckerMassPreviewCapture
+      :assets="assets"
+      :index-content="indexContent"
+      :previews="PREVIEWS_AMOUNT" />
+  </div>
+</template>
+<script lang="ts" setup>
+import { AssetMessage } from '../types'
+
+const PREVIEWS_AMOUNT = 12
```

Comment:
kinda hidden away, and very likely to get changed in the future / by user

i.e make it props with default value